rebirthdb/_export.py

```diff
@@ -335,11 +335,7 @@ def export_table(db, table, directory, options, error_queue, progress_info, sind
 
             except (errors.ReqlTimeoutError, errors.ReqlDriverError):
                 # connection problem, re-setup the cursor
-                try:
-                    cursor.close()
-                except Exception:
-                    pass
-
+                cursor.close()
```

Comment:
Good point. Will fix it on konday with the reported quality regressions.

Reply:
@grandquista Maybe I overlooked something but what exception could be raised by ‘close()’?

Reply:
`close` runs `run_query` which can raise several `ReQLError`s as well as `OSError`. I would expect all the `ReQLError` cases are recoverable. The `OSError`s are questionable since they likely mean the network has a more permanent problem.

Reply:
Can we agree on only catch reql errors

Reply:
I agree with that.